MAUI/Cartesian-Charts/Axis/Axislabels.md

```diff
@@ -166,4 +166,79 @@ chart.XAxes.Add(primaryAxis);
 
 {% endhighlight %}
 
+{% endtabs %}
+
+## Smart Axis Labels
+
+Axis labels may overlap with each other based on chart dimensions and label size. The [LabelsIntersectAction]() property of axis is used to avoid overlapping of axis labels. The default value of the LabelsIntersectAction is `None`; other available values are `MultipleRows`, `Hide`, and `Wrap`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+. . .
+    <chart:SfCartesianChart.XAxes>
+       <chart:CategoryAxis LabelsIntersectAction="MultipleRows" />
+    </chart:SfCartesianChart.XAxes>
+
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+. . .
+CategoryAxis primaryAxis = new CategoryAxis()
+{
+    LabelsIntersectAction = AxisLabelsIntersectAction.MultipleRows,
+};
+ chart.XAxes.Add(primaryAxis);
+ 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+**Wrap**
+
+Using the [MaxWidth]() property, we can set the wrap width of the axis labels. It also provides support to customize the rendering position of the axis labels using the [WrappedLabelAlignment]() property. The default value is `start`; other available values are `center` and `end`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+. . .
+    <chart:SfCartesianChart.XAxes>
+      <chart:CategoryAxis LabelsIntersectAction="Wrap">
+         <chart:CategoryAxis.LabelStyle >
+             <chart:ChartAxisLabelStyle  MaxWidth="20" WrappedLabelAlignment="Center" />
```

Comment:
@Sowndharya1911 as MaxWidth & WrappedLabelAlignment API exposed in axis LabelStyle class, update it under LabelStyle content. For wrap option, we can mention as notes

Reply:
@VimalaThirumalaiKumarSF3739  Addressed the suggestions